### PR TITLE
I145 release notes v12

### PR DESCRIPTION
**Pull request by Tereza-Pytelova**

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,4 +1,63 @@
...
+
+### New Features
+ - Support for cloud functions was added for deployment and testing
+ - WAW now supports [pattern entities](/doc/WAW_entities_doc.md#patterns) and [contextual entities](/doc/WAW_entities_doc.md#contextual-entities) (csv to json direction only)
+ - We employ `logging` module for logging
+ - Using [workspace_addjson.py](/scripts/workspace_addjson.py) script you are now able to add custom json structures to specified location in the workspace json
+ - It is now possible to deploy workspace by its name (by setting `workspace_name_unique` to true)
+ - [Dialog testing script](/scripts/evaluate_tests.py) now outputs `junit.xml` file with test results
+ - [requirements.txt](/requirements.txt) and [requirements_dev.txt](/requirements_dev.txt) were added to simplify installation process
+ - Configuration files was splitted to three ones: [common.cfg](/example/en_app/common.cfg) (common project settings), [private.cfg](/example/en_app/private.cfg.template) (credentials - not to push) and [builg.cfg](/example/en_app/build.cfg) (setting specific for the build - usualy not to push)
```
Comment (michal-mylek) [inline release_notes.md]: Change proposals:
- "Sample configuration file was split into three: ..."
- private.cfg.template) (filled-in template contains credentials, not to be pushed)
- build.cfg) (settings specific for concrete build environment, usually not to be pushed)

File: release_notes.md [hunk 2]
```diff
@@ -1,4 +1,63 @@
...
+
+#### WAW xml format
+
+ - Following node tags were added into [WAW xml format](/data_spec/dialog_schema.xml) `disabled`, `metadata`, `type` and digression-connected tags
```
Comment (michal-mylek) [inline release_notes.md]: is digression-connected missing ``?

File: release_notes.md [hunk 3]
```diff
@@ -1,4 +1,63 @@
...
+ - `event_name` tag was added for handlers
+ - Nodes have new attribute `title`
+ - In `goto` structure, there is a new tag `next_step` defaulting to `jump_to`
+ - It is possible to add [`scope`](/doc/WAW_dialog_doc.txt) attribute to most of the nodes. Scope can be used for creating customized dialogs from a default one.
+ - Output structure was updated accordingly to WA changes to contain `generic` tag
+
+#### T2Cformat
+ - [T2C format](/doc/T2C_doc.md) now supports folders in a form of an extra channel
+ - Options (buttons) in T2C are translated to a format which is understood by clientv2 (it is not yet compliant with response types)
+ - Foldables are supported by T2C. Foldables are texts present in a short and a long form. If client supports rendering of them (clentv2 does), user can switch between these two forms by clicking on the text.
```
Comment (michal-mylek) [inline release_notes.md]: clientv2 typo?
"If client supports their rendering,..."

File: release_notes.md [hunk 4]
```diff
@@ -1,4 +1,63 @@
...
+ - Reverse order of imported nodes was fixed and nodes are imported to a specified position, not at the end of a node list
+ - String formatting issues were fixed
+
+### WAW CI
+ - Nightly builds are set up for `devel` and `master` branches
+ - Fork PR builds are enabled
+ - Artifactory cleanup takes place on each run
+ - CI scripts were rewritten to `pytest`
+ - Unit testing is enabled for scripts and will be part of all future features
+ - `flake8` tests are now part of the CI process
```
Comment (michal-mylek) [inline release_notes.md]: Clarify, whether tests are actually being run, or this is just informative (~on TO's behalf)

Comment (Tereza-Pytelova) [reply release_notes.md]: Done.

File: release_notes.md (hunk 3 above)
Comment (Tereza-Pytelova) [reply release_notes.md]: Fixed.

File: release_notes.md (hunk 2 above)
Comment (Tereza-Pytelova) [reply release_notes.md]: It stands for all digression connected tags, but I understand that it is quite misleading. I listed all of them instead.

File: release_notes.md (hunk 1 above)
Comment (Tereza-Pytelova) [reply release_notes.md]: Changed, thank you!!

**Pull request merged**